carball/analysis/stats/boost/boost.py

```diff
@@ -33,10 +41,34 @@ def get_player_boost_usage(player_dataframe: pd.DataFrame) -> np.float64:
 
     @staticmethod
     def get_player_boost_usage_max_speed(player_dataframe: pd.DataFrame) -> np.float64:
-        """TODO: do this"""
         _diff = -player_dataframe.boost.diff()
```

Comment:
This should use `get_player_boost_usage`, no?
Or in fact just take the above `proto_boost.boost_usage` as the second parameter (`boost_usage: pd.Series`)